### PR TITLE
Fix package json

### PR DESCRIPTION
**Pull request by 0xR**
Adds graphql as a peerDependency. Avoids issues with duplicate instances of GraphQL.
Adds repository field.
Replaces prePublish with a prepare scripts.

### CODE REVIEW

**Comment by yarax** [comment]
Nice, thanks. What's the goal of replacing prepublish?

**Comment by 0xR** [comment]
prepublish is depracated and building should happen in the new `prepare` script:
https://github.com/npm/npm/issues/10074

**Comment by yarax** [comment]
I see, good to know!

File: package.json [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
   "version": "1.3.0",
   "author": "Roman Krivtsov",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yarax/swagger-to-graphql.git"
+  },
   "bin": "./bin/swagger2graphql",
   "dependencies": {
     "babel-runtime": "^6.25.0",
-    "graphql": "^0.12.3",
```
Comment (Darby101) [inline package.json:8]: {
  "scripts": {
    "precompress": "{{ executes BEFORE the `compress` script }}",
    "compress": "{{ run command to compress files }}",
    "postcompress": "{{ executes AFTER `compress` script }}"
  }
}

File: package.json [hunk 2]
```diff
@@ -45,7 +52,7 @@
   "scripts": {
     "build": "babel src -d lib",
     "lint": "eslint src/",
-    "prepublish": "npm run build",
```
Comment (Darby101) [inline package.json:48]: package.json

**Review by Darby101 — CHANGES_REQUESTED** [review]

**Comment by Joe1the2Creator3** [comment]
{
  "scripts": {
    "precompress": "{{ executes BEFORE the `compress` script }}",
    "compress": "{{ run command to compress files }}",
    "postcompress": "{{ executes AFTER `compress` script }}"
  }
}

**Review by bdstar81 — APPROVED** [review]

**Pull request merged**